SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslator.cpp

```diff
@@ -38,14 +38,14 @@ Worker_EntityId SpatialVirtualWorkerTranslator::GetServerWorkerEntityForVirtualW
 
 void SpatialVirtualWorkerTranslator::ApplyVirtualWorkerManagerData(Schema_Object* ComponentObject)
 {
-	UE_LOG(LogSpatialVirtualWorkerTranslator, Log, TEXT("(%d) ApplyVirtualWorkerManagerData"), LocalVirtualWorkerId);
```

Comment:
As per the ticket, can you also move change some startup logs to being Verbose inside VirtualWorkerTranslationManager such as:
```
LogSpatialVirtualWorkerTranslationManager: Processing ServerWorker Entity query response
LogSpatialVirtualWorkerTranslationManager: Waiting for all virtual workers to be assigned before publishing translation update.
LogSpatialVirtualWorkerTranslationManager: Sending query for WorkerEntities
```